amnestywebsite/humanity-theme
# Fix output of default footer content

Use existing site configuration where possible.
This will be deprecated in the future, but
is necessary for the interim, so that we don't
have to re-build every site's footer

**Steps to test**:
1. whilst on version `v1.2.0` of the theme (`git checkout tags/v1.2.0`), visit a non-English site
2. look at the footer - the contents should reflect the settings in Theme Options -> Footer
3. checkout `v2.0.0` of the theme (`git checkout tags/v2.0.0`)
4. reload the page and look at the footer again
5. you'll see that the footer becomes the English .org footer
6. checkout this PR
7. reload the page and look at the footer again
8. it should reflect the original footer's state
9. edit the footer in the site editor
10. your changes should reflect on the site
